website/docs/r/cis_waf_package.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_waf_package"
+sidebar_current: "docs-ibm-resource-cis-waf-package"
+description: |-
+  Provides a IBM CIS WAF Package resource.
+---
+
+# ibm_cis_waf_package
+
+Provides a IBM CIS WAF Package resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to change WAF Package settings of a domain of a CIS instance
+
+## Example Usage
+
+```hcl
+# Change WAF Package settings of the domain
+
+resource "ibm_cis_waf_package" "waf_package" {
+	cis_id      = data.ibm_cis.cis.id
+	domain_id   = data.ibm_cis_domain.cis_domain.domain_id
+	package_id  = "c504870194831cd12c3fc0284f294abb"
+	sensitivity = "low"
+	action_mode = "block"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `cis_id` - (Required,string) The ID of the CIS service instance.
+- `domain_id` - (Required,string) The ID of the domain to change TLS settings.
+- `package_id` - (Required, string) The WAF package ID. This can not be modified.
```

Comment:
package_id is a force _new argument

Reply:
do you want me to add `force new` word in this line?